### PR TITLE
+nim-lang.org

### PR DESCRIPTION
**Pull request by ivabus**
Init at v1.6.14

### CODE REVIEW

**Comment by ivabus** [comment]
Oh, they are waiting for gcc on linux, fix in the way

**Comment by ivabus** [comment]
Ready to merge, review pls

File: projects/nim-lang.org/package.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+dependencies:
+  gnu.org/gcc: '*'
+  pcre.org: '*'
+  openssl.org: '*'
+
+build:
+  dependencies:
+    gnu.org/gcc: '*'
+    pcre.org: '*'
+    openssl.org: '*'
```
Comment (mxcl) [inline projects/nim-lang.org/package.yml]: if these are the same then you don't need to specify them again in the `build` node, however are these actually also runtime deps?

File: projects/nim-lang.org/package.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+distributable:
+  url: https://nim-lang.org/download/nim-{{ version }}.tar.xz
+  strip-components: 1
+
+# FIXME: test more versions
+versions:
+  - v1.6.14
```
Comment (mxcl) [inline projects/nim-lang.org/package.yml]: is there are URL that we can scrape? /cc @jhheider

**Review by mxcl — CHANGES_REQUESTED** [review]

File: projects/nim-lang.org/package.yml (hunk 1 above)
Comment (ivabus) [reply projects/nim-lang.org/package.yml]: Yes, these are actually runtime deps. Nim compiles to C++, so it needs C++ compiler to run code (LLVM doesn't work properly when compiling Nim, so I turned on gcc, which works fine).

File: projects/nim-lang.org/package.yml (hunk 2 above)
Comment (ivabus) [reply projects/nim-lang.org/package.yml]: I think we can use GitHub tags. (Commited)

**Comment by jhheider** [comment]
Looking good, @ivabus ; sorry for the delay in addressing.

**Comment by ivabus** [comment]
Yay!

**Pull request merged**